dependencies.py

```diff
@@ -124,6 +126,26 @@ def _install_dependencies(dependencies, target_dir):
         print("Installed %s into %s" % (dependency.name, target_dir))
 
 
+def _get_python_interpreter():
+    """Return the path to the python3 interpreter.
+
+    Under linux sys.executable is set to the python3 interpreter used by Qgis.
+    However, under Windows/Mac this is not the case and sys.executable refers to the
+    Qgis start-up script.
+    """
+    interpreter = None
+    executable = sys.executable
+    head, tail = os.path.split(executable)
+    if tail == "python3":
+        interpreter = executable
+    elif tail == "qgis-ltr-bin.exe":
```

Comment:
Iets minder gevoelig voor veranderingen ("qgis-bin.exe" als iemand het in een niet-ltr probeert bijvoorbeeld):

```suggestion
    elif "qgis" in filename:
```